helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -127,6 +127,14 @@ controller:
 image:
   registry: quay.io
 
+initContainer:
+  name: sysctl
+  image:
+    repository: giantswarm/alpine
+    tag: 3.9-giantswarm-sysctl
```

Comment:
```suggestion
    tag: 3.10-giantswarm-sysctl
```
Today 3.10-giantswarm-sysctl appeared https://quay.io/repository/giantswarm/alpine?tab=tags

I think there is some bug in retagger logic, for this case where pattern is being used and there are two custom images defined https://github.com/giantswarm/retagger/blob/master/images.yaml#L4-L13

With enough of (nightly) runs it will eventually get all tags synced. There's higher prio critical stuff. Still, not to forget about this potential bug, will create an issue for sig-releng to have a look in retagger.